datahub-web-react/src/app/search/autoComplete/quickFilters/QuickFilter.tsx

```diff
@@ -8,7 +8,7 @@ import { getQuickFilterDetails } from './utils';
 import { ANTD_GRAY } from '../../../entity/shared/constants';
 import analytics, { Event, EventType } from '../../../analytics';
 
-const QuickFilterWrapper = styled(Button)<{ isSelected: boolean }>`
+const QuickFilterWrapper = styled(Button)<{ selected: boolean }>`
```

Comment:
curious about isSelected > selected.

Just for brevity? Or is this a consistency thing you noticed?

Reply:
Would not have normally changed this but I was getting js console warnings about isSelected not being a valid name style for a dom node.